cmd/release-controller/http_helper.go

```diff
@@ -873,3 +879,11 @@ func checkReleasePage(page *ReleasePage) {
 		}
 	}
 }
+
+func dashboardsJoin(dashboards []Dashboard) string {
+	boards := []string{}
+	for _, d := range dashboards {
+		boards = append(boards, fmt.Sprintf("<a href=\"%s\">%s</a>", d.Path, d.Name))
```

Comment:
you need to html escape name

Reply:
I escaped both the path and the name just in case and I also corrected the function that this function was based from.